plugins/ladspa_browser/ladspa_browser.cpp

```diff
@@ -48,7 +48,7 @@ extern "C"
 Plugin::Descriptor PLUGIN_EXPORT ladspabrowser_plugin_descriptor =
 {
 	STRINGIFY( PLUGIN_NAME ),
-	"LADSPA Plugin Browser",
+	QT_TRANSLATE_NOOP( "LADSPA Plugin Browser" ),
```

Comment:
You need to specify the context here, like `QT_TRANSLATE_NOOP( "pluginBrowser", "LADSPA Plugin Browser" )` or `QT_TRANSLATE_NOOP( "MainWindow", "LADSPA Plugin Browser" )`. Personally I don't like this approach...

Reply:
@PhysSong : followed you advice, I choose MainWindow rather than pluginBrowser since it's a menu item on the main windows, but I could be wrong. Am I done better now?